.github/workflows/linter.yml

```diff
@@ -0,0 +1,19 @@
+on: [push, pull_request]
+name: linter
+
+jobs:
+  lint:
+    strategy:
+      matrix:
+        go-version: [1.x]
+        platform: [ubuntu-latest]
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: golangci-lint
+      uses: golangci/golangci-lint-action@v2
```

Comment:
```suggestion
      uses: golangci/golangci-lint-action@5c56cd6c9dc07901af25baab6f2b0d9f3b7c3018 #v2.5.2
```

Reply:
Google requires using exact SHAs for third-party actions.

Reply:
Done :)!